components/raftstore/src/store/fsm/peer.rs

```diff
@@ -2112,13 +2132,16 @@ where
             );
         }
         let is_initialized = self.fsm.peer.is_initialized();
+        self.ctx.perf_context.start_observe();
```

Comment:
How about moving `perf_context` modifications into `destroy`? I worry about someone can forget it again.

Reply:
Done. I realize that the `perf_context` here will record the kv db which is not right because it's raftstore's perf context.(We should add a new `perf_context` to record kv db metrics here.